src/util/GeoSparqlHelpers.h

```diff
@@ -144,10 +144,10 @@ class WktGeometricRelation {
       [[maybe_unused]] const std::optional<GeoPoint>& geoLeft,
       [[maybe_unused]] const std::optional<GeoPoint>& geoRight) const {
     AD_THROW(
-        "Geometric relations via the `geof:sf...` functions are not yet "
-        "implemented in QLever. Please refer to the custom `SERVICE qlss:` "
-        "with algorithm `qlss:libspatialjoin` for now. More details can be "
-        "found on the QLever Wiki.");
+        "Geometric relations via the `geof:sfIntersects` ... functions are "
+        "currently only implemented for queries of the form "
+        "`FILTER(geof:sfIntersect(?a, ?b))`, ... More details on GeoSPARQL "
+        "support can be found on the QLever Wiki.");
```

Comment:
Is it correct that this might happen in strange cases where the query planner (for some reason or the other) can't apply the substitute spatial join, and therefore we get this misleading error message?